### ISSUE TITLE
RFC: Add some rule customisation 

### ISSUE THREAD

**Issue by lyrixx** [VincentLanglet/Twig-CS-Fixer#207]
Hello, 

First, congrats for this tool! I installed it, and ran it in a couple of seconds!

I like the updated CS, but there are two "blockers" for me:

```diff
- {{ form_label(form.markdownName, null, { label_attr: { class: 'd-md-none' }}) }}                                                                                             
+ {{ form_label(form.markdownName, null, {label_attr: {class: 'd-md-none'}}) }}
```

I prefer the old version, and bellow too.

```diff
- <p class="c-partner-info__address">{{ location.address | nl2br }}</p>
+ <p class="c-partner-info__address">{{ location.address|nl2br }}</p>
```

I read the doc, but I cannot find a way to disable/configure theses rules

**Comment by lyrixx** [comment]
https://github.com/VincentLanglet/Twig-CS-Fixer/issues/207#issuecomment-2057059508 => oh, this is a very valid argument! I think I'll change my point of view!

https://github.com/VincentLanglet/Twig-CS-Fixer/issues/207#issuecomment-2057100475 => indeed. I prefer with space before and after `{` or `}`

**Comment by lyrixx** [comment]
Awesome 👍🏼

**Comment by VincentLanglet** [comment]
> Awesome 👍🏼

Can you confirm that https://github.com/VincentLanglet/Twig-CS-Fixer/pull/208 works for you @lyrixx ?

**Comment by lyrixx** [comment]
OHOH, work very well. 👏🏼 

---

Side note: I think you could improve the documentation about rule configuration.

IMHO, you should talk `TwigCsFixer` since it the default:

```
<?php

$ruleset = new TwigCsFixer\Ruleset\Ruleset();
$ruleset->addStandard(new TwigCsFixer\Standard\TwigCsFixer());
// Add you rules here

$config = new TwigCsFixer\Config\Config();
$config->setRuleset($ruleset);

return $config;
```

And another topic about rule configuration

```
$ruleset = new TwigCsFixer\Ruleset\Ruleset();
$ruleset->addStandard(new TwigCsFixer\Standard\TwigCsFixer());
$ruleset->overrideRule((new TwigCsFixer\Rules\Punctuation\PunctuationSpacingRule(
    ['}' => 1],
    ['{' => 1],
)));

$config = new TwigCsFixer\Config\Config();
$config->setRuleset($ruleset);

return $config;
```

**Comment by VincentLanglet** [comment]
Does https://github.com/VincentLanglet/Twig-CS-Fixer/pull/209/files would be helpful ?
It's not easy to write doc ^^'

**Comment by VincentLanglet** [comment]
Released in 2.7.0
